build.gradle

```diff
@@ -229,7 +229,7 @@ dependencies {
 
   implementation group: 'com.azure.spring', name: 'spring-cloud-azure-starter-active-directory', version: '5.17.1'
   implementation group: 'org.postgresql', name: 'postgresql', version: '42.7.4'
-  implementation group: 'com.github.hmcts', name: 'pip-data-models', version: '2.1.32', {
+  implementation group: 'com.github.hmcts', name: 'pip-data-models', version: '2.1.34', {
```

Comment:
Chris has merged in new pip-data-models PR. This needs to be updated to 2.2.0

Reply:
done